### PR TITLE
Update NfcGuideView resources

### PR DESCRIPTION
**Pull request by Franzaine**

### CODE REVIEW

**Comment by Franzaine** [comment]
@petter-fidesmo ready for review. I'll make sure to also get +1 on the drawings.

**Comment by Franzaine** [comment]
Miguel :+1: the drawings

**Comment by Franzaine** [comment]
@slomo ready for review

**Comment by Franzaine** [comment]
@petter-fidesmo

File: android/src/main/java/nordpol/ui/NfcGuideView.java [hunk 1]
```diff
@@ -100,6 +100,8 @@ public NfcGuideView(Context context, AttributeSet attrs) {
         case 1:
             mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_card_black));
             break;
+	case 2:
+	    mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_usb_black));
```
Comment (petter-fidesmo) [inline android/src/main/java/nordpol/ui/NfcGuideView.java]: @Franzaine It's best practice to also have the break; after the last clause.

**Comment by petter-fidesmo** [comment]
@Franzaine :+1:

**Pull request merged**